BanchoSharp/Multiplayer/MultiplayerMods.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace BanchoSharp.Multiplayer;
+
+// It's important that these are named the same way Bancho names them in IRC, 
+// in order for them to get registered correctly.
+[Flags]
+public enum Mods
+{
+    None = 0,
+    Freemod = 1 << 0,
+    Easy = 1 << 1,
+    NoFail = 1 << 2,
+    HalfTime = 1 << 3,
+    HardRock = 1 << 4,
+    SuddenDeath = 1 << 5,
+    Perfect = 1 << 6,
+    DoubleTime = 1 << 7,
+    Nightcore = 1 << 8,
+    Hidden = 1 << 9,
+    FadeIn = 1 << 10,
+    Flashlight = 1 << 11,
+    Relax = 1 << 12,
+    Autopilot = 1 << 13,
+    SpunOut = 1 << 14
+}
+
+public static class ModsExtensions
+{
+
+    /// <summary>
+    /// Returns a string with a shorter format, example: "HDDT"
+    /// </summary>
+    public static string ToShortString(this Mods mods, bool showFreemod = true)
+    {
+        var ret = "";
```

Comment:
I would rather a `StringBuilder` object be used here, string appending takes up resources.